View/Components/ID3ArtistsTagConverter.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using System.Windows.Data;
+
+namespace View.Components
+{
+    public class ID3ArtistsTagConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            var artists = value as string[];
+
+            return string.Join(" ", artists ?? Array.Empty<string>());
+        }
+
+        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            throw new NotImplementedException();
```

Comment:
Ennuh, als je m toch niet gebruikt. Kan deze functie dan niet gewoon weg?

Reply:
IValueConverter wil ConvertBack er in hebben zitten